src/index.html

```diff
@@ -12,16 +12,436 @@
       href="https://fonts.gstatic.com"
     />
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@400;500;700&display=swap"
```

Comment:
The Google Fonts API link includes both 'Poppins' and 'Roboto'. Verify if both fonts are necessary to avoid redundancy.